build.gradle.kts

```diff
@@ -58,7 +58,7 @@ tasks {
       "buildMappingServiceApiModel",
       "buildSentencingAdjustmentsApiModel",
       "buildCourtSentencingApiModel",
-      "buildAlertApiModel",
+      "buildAlertsApiModel",
```

Comment:
make it easier to translate between project and model build otherwise need a map of these as well